src/core/ConfigManager.cpp

```diff
@@ -139,7 +139,8 @@ void ConfigManager::upgrade()
 	}
 
 	// Runs all necessary upgrade methods
-	std::for_each( UPGRADE_METHODS.begin() + m_configVersion, UPGRADE_METHODS.end(),
+    std::size_t max = std::min((std::size_t)m_configVersion, UPGRADE_METHODS.size());
```

Comment:
spaces indention here instead of tabs :)

Reply:
Solved.